test/app.js

```diff
@@ -151,8 +152,15 @@ describe('generator-camel:app', function () {
     });
 
     it('Should create pom.xml with default content', function () {
-      assert.fileContent('pom.xml', new RegExp('<groupId>' + basicProps.package + '</groupId>') );
-      assert.fileContent('pom.xml', new RegExp('<artifactId>' + basicProps.name + '</artifactId>') );
+      assert.fileContent('pom.xml', new RegExp('<groupId>' + basicProps.package + '</groupId>'));
+      assert.fileContent('pom.xml', new RegExp('<artifactId>' + basicProps.name + '</artifactId>'));
+    });
+  });
+
+  describe('Should properly scaffold with default config for Java DSL', function () {
+    it('utilities should pass basic tests', function () {
```

Comment:
I think it would be best to create two different test methods.
something like:
- Check validation package utility pass for correct package name
- Check validation package utility detect invalid package name